### DISCUSSION TITLE
Customise the right column to include recent posts and/or small bio

### DISCUSSION THREAD

**Discussion by jezkerwin** [nunocoracao/blowfish#117] [Q&A] [answered]
When viewing a post is it possible to customise the right column to include other items like perhaps tags, recent posts or even a box that contains a small bio of the author in addition or replacing the table of contents?

**Comment by nunocoracao** [comment, marked as answer]
Hello @jezkerwin, with the current UI and configs not out-of-the-box. But Hugo does a great job at allowing people to extend the basis stuff of whatever theme. You can create an overlays folder that will override the ones on the theme to implement that experience. Alternatively, you can propose a new feature in the issues section. Thanks again.
